README.md

```diff
@@ -277,6 +277,48 @@ jobs:
         credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
 ```
 
+### Authenticating to Container Registry and Artifact Registry
+
+This example demonstrates authenticating to Google Container Registry (GCR) or
+Google Artifact Registry (GAR). The most common way to authenticate to these
+services is via a gcloud docker proxy. However, you can authenticate to these
+registries directly using the `auth` action:
+
+-   **Username:** `oauth2accesstoken`
+-   **Password:** `${{ steps.auth.outputs.access_token }}`
+
+You must set `token_format: access_token` in your Action YAML. Here are a few
+examples:
+
+```yaml
+jobs:
+  job_id:
+    steps:
+    - uses: 'actions/checkout@v2'
+
+    - id: 'auth'
+      name: 'Authenticate to Google Cloud'
+      uses: 'google-github-actions/auth@v0'
+      with:
+        token_format: 'access_token'
+        # Either user Workload Identity Federation or Service Account Keys. See
+        # above more more examples
+
+    # This example uses the docker login action
+    - uses: 'docker/login-action@v1'
+      with:
```

Comment:
Don't we also need to specify the registry?
```suggestion
      with:
        registry: gcr.io # or <location>-docker.pkg.dev
```

Reply:
Done